src/Views/ContactUs.js

```diff
@@ -77,19 +87,45 @@ export default function ContactUs() {
 	const handleMessageBodyChange = (e) => {
 		setMessage(e.target.value);
 	};
+	const handleMessageSent = (e) => {
+		setMessageSent(e.target.value);
+	};
 
 	const fieldWidth = "100%";
 
-	return (
+	return (messageSent ?
+		<>
+		<Container maxWidth="md">
+			<Typography
+			style={{ marginBottom: "25px", marginTop: "40px" }}
+			variant="h2" className={classes.header}>
+			Contact Us
+			</Typography>
+			<Card variant="outlined" className={classes.root}>
+				<img src="https://pluspng.com/img-png/green-tick-png-green-tick-transparent-png-600.png" width="5%" heigth = "5%"/>
```

Comment:
This works, but it relies on another another API to get the image. Download the picture and save it in the files locally. We will work out how to load images faster later.